tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "noImplicitUseStrict": true,
     "outDir": "local_compiled_js",
     "rootDir": ".",
+    "sourceMap": true,
```

Comment:
What does this do? More specifically, does this increase the size of files in prod mode?

Reply:
This isn't needed any more, so I'm removing it.